lib/testers/rule-tester.js

```diff
@@ -33,9 +33,9 @@ function verify(ruleName, rule, testCases) {
 
     describe('should lint valid', function() {
 
-      testCases.valid.forEach(({ moddleElement }, idx) => (
+      testCases.valid.forEach(({ moddleElement, it: _it }, idx) => (
```

Comment:
I like this, very useful. Could we add documentation in the README.md how to use this (I would not know right away?)

Reply:
The way to go would be type definitions for me. Or a dedicated slot in an example project.

We do not have any documentation regarding plug-in testing. Adding some is out of scope of this PR.

Reply:
I could think about something along [these lines](https://github.com/bpmn-io/bpmnlint-plugin-example/pull/2).